cmip6_downscaling/methods/common/tasks.py

```diff
@@ -161,6 +162,25 @@ def pyramid(
     return target
 
 
+@task
+def regrid(source_path: UPath, target_grid_path: UPath) -> UPath:
+
+    target = intermediate_dir / "regrid" / source_path / target_grid_path
```

Comment:
I think you are going to need to do something like this: 

```suggestion
    target = intermediate_dir / "regrid" / source_path.path.replace('/', '_') + target_grid_path.path.replace('/', '_')
```